setup.py

```diff
@@ -1,99 +0,0 @@
-# Require setuptools -- distutils does not support install_requires
```

Comment:
Whoa... poetry doesn't need this at all?

Reply:
Yes. `python -m build` installs setuptools in a venv if required.